src/Wallabag/CoreBundle/Resources/views/themes/baggy/Entry/entry.html.twig

```diff
@@ -17,6 +17,7 @@
             <li><a title="{{ 'entry.view.left_menu.set_as_favorite'|trans }}" class="tool icon icon-star {% if entry.isStarred == 0 %}fav-off{% else %}fav{% endif %}" href="{{ path('star_entry', { 'id': entry.id }) }}"><span>{{ 'entry.view.left_menu.set_as_favorite'|trans }}</span></a></li>
             <li><a id="nav-btn-add-tag" title="{{ 'entry.view.left_menu.add_a_tag'|trans }}"><span>{{ 'entry.view.left_menu.add_a_tag'|trans }}</span></a></li>
             <li><a title="{{ 'entry.view.left_menu.delete'|trans }}" class="tool delete icon icon-trash" href="{{ path('delete_entry', { 'id': entry.id }) }}"><span>{{ 'entry.view.left_menu.delete'|trans }}</span></a></li>
+            {% if craue_setting('share_public') %}<li><a href="{{ path('share', {'uuid': entry.uuid }) }}" target="_blank" class="tool public" title="public"><span>public</span></a></li>{% endif %}
```

Comment:
`public` needs to be translated.
Also I think _public link_ should be a better label.